app/api/domains/modules/endpoints/list_module_objects_endpoint.py

```diff
@@ -54,6 +60,9 @@ def get_list_module_objects_endpoint(
         actions=actions,
     )
 
+    module_objects = [module_object for module_object, _ in paginated_result.items]
+    models = module_objects_to_models_caster.cast(module_objects, context.model_map)
```

Comment:
Wil je waar mogelijk de types toevoegen zodat de toekomstige ons meteen weten wat de variabele is